src/domains/wallet-app/api/npg/v1/_base_policy.xml.tpl

```diff
@@ -0,0 +1,15 @@
+<policies>
+    <inbound>
+      <base />
+      <set-backend-service base-url="https://${hostname}/pagopa-wallet-notification-service" />
```

Comment:
```suggestion
      <set-backend-service base-url="https://${hostname}/pagopa-wallet-service" />
```

Should it be the same backend service url as the one exposed for the wallet?

Reply:
I push this different url to have clarification about something not clear for me.
This api is declared in a new api group in wallet project, so my doubt is if we have to maintain same backend service url or it should be different.

Reply:
Solved my doubt, I'll push your suggestion